src/router.js

```diff
@@ -18,6 +18,11 @@ export default new Router({
       name: 'login',
       component: Login
     },
+    {
+      path: '/auth',
```

Comment:
`/`

Reply:
I don think it should be `/`. The Trello says two things:
1) When the user goes to `/`, must be redirected to `/login`

Reply:
Yes, there was a mistake with the trello bullet. It's already been corrected